code/test/general.js

```diff
@@ -0,0 +1,945 @@
+const TestUtil = require('./util')
```

Comment:
Are some of these tests repeating tests that are already in other files (e.g. transfer, token transfer, tracking)? If so, shouldn't they be removed? Also, to avoid more repetitions, and to make scope of testing clear for each test, can all tests in this file be moved to files designated for specific purposes? We can use a .md file to track coverage of each operation, instead of a `general` file that repeats tests already performed in each scope